recipes/scrapy/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "Scrapy" %}
+{% set version = "1.1.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "fab9c5074bc484add855d2d072ff26fdc0ba5344858cc68178f4f4f39a6f2cce" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  # Skip win+py3k as the build as the framework is not 100% functional.
+  skip: True  # [win and py3k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - scrapy = scrapy.cmdline:execute
+  number: 0
+
+# The requirements below shall match the requirements from the target package
+# version (check setup.py for changes).
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - twisted >=10.0.0
+    - w3lib >=1.15.0
+    - queuelib
+    - lxml
+    - pyopenssl
+    - cssselect >=0.9
+    - six >=1.5.2
+    - parsel >=0.9.3
+    - pydispatcher >=2.0.5
+    - service_identity
+    - pywin32  # [win]
+
+test:
+  imports:
+    - scrapy
+    - scrapy.commands
+    - scrapy.contracts
+    - scrapy.contrib
+    - scrapy.contrib.downloadermiddleware
+    - scrapy.contrib.exporter
+    - scrapy.contrib.linkextractors
+    - scrapy.contrib.loader
+    - scrapy.contrib.pipeline
+    - scrapy.contrib.spidermiddleware
+    - scrapy.contrib.spiders
+    - scrapy.contrib_exp
+    - scrapy.contrib_exp.downloadermiddleware
+    - scrapy.core
+    - scrapy.core.downloader
+    - scrapy.core.downloader.handlers
+    - scrapy.downloadermiddlewares
+    - scrapy.extensions
+    - scrapy.http
+    - scrapy.http.request
+    - scrapy.http.response
+    - scrapy.linkextractors
+    - scrapy.loader
+    - scrapy.pipelines
+    - scrapy.selector
+    - scrapy.settings
+    - scrapy.spidermiddlewares
+    - scrapy.spiders
+    - scrapy.utils
+    - scrapy.xlib
+    - scrapy.xlib.tx
+
+  commands:
+    # Ideally, we will run tests here but we require to provide the tests
+    # requirements first.
```

Comment:
I'm a little confused by this comment. What is this suppose to mean? Do we need certain requirements to run the tests? If so, it is easy to add them. Or is there something else required here?